subsys/net/ip/tcp.c

```diff
@@ -848,6 +848,37 @@ static int ip_header_add(struct tcp *conn, struct net_pkt *pkt)
 	return -EINVAL;
 }
 
+static int set_tcp_nodelay(struct tcp *conn,
+				const void *value, size_t len)
+{
+	if (len != sizeof(int)) {
+		return -EINVAL;
+	}
+
+	int no_delay_int = *(int *)value;
```

Comment:
Minor nit, we try to introduce the variables at the beginning of the block.

Reply:
Fixed